### PR TITLE
more ga tracking

### PR DESCRIPTION
**Pull request by AWare**
track the additional required fields for ga
@ajosephides @paulbrown1982

### CODE REVIEW

**Comment by ajosephides** [comment]
@AWare - will you also add the internal campaign code dimension (12). Something I might need to run through with you.

File: frontend/app/views/joiner/thankyou.scala.html [hunk 1]
```diff
@@ -51,7 +51,12 @@ <h2 class="page-section__headline">@title</h2>
 }
 
 @main(title) {
-
+<script type="text/javascript">
+   window.productData = {
```
Comment (paulbrown1982) [inline frontend/app/views/joiner/thankyou.scala.html]: Maybe add this to window.guardian.productData instead? That would be more consistent with subs.

Comment (AWare) [inline frontend/app/views/joiner/thankyou.scala.html]: window.guardian isn't instantiated until the foot.

**Comment by AWare** [comment]
Changed as per @paulbrown1982  and @ajosephides

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**